src/server/services/wms/qgswmsgetserversettings.cpp

```diff
@@ -25,7 +25,7 @@
 #include "pg_config.h"
 #include "spatialite.h"
```

Comment:
You'll need to `<>` all which are not `qgs.....h`

Reply:
Thank you @m-kuhn!